src/app/dataset-create/dataset-create.component.html

```diff
@@ -44,7 +44,7 @@ <h1 class="title">Create a new dataset</h1>
             <label for="kind-derivative" class="mt-2 d-flex">
                 <input name="kind" type="radio" value="derivative" formControlName="kind" />
```

Comment:
Please add the `id` same as the attribute `for` on the label
```suggestion
                <input name="kind" type="radio" id="kind-derivative" value="derivative" formControlName="kind" />
```